google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/it/ITBigQueryWriteManualClientTest.java

```diff
@@ -28,11 +28,14 @@
 import com.google.cloud.bigquery.storage.test.Test.*;
```

Comment:
I think we should also add a unit test, just the simple case, so that the issue can be discovered by running unit test. In unit test, the update schema can be mimiced by directly return a mock response with table schema.

Reply:
Will add the unit test in a separate PR since I'm trying to resolve some intelliJ issue related to `import com.google.cloud.bigquery.storage.test.Test.*;` not being recognized.

Reply:
Resolved IDE issue. [Unit test](https://github.com/googleapis/java-bigquerystorage/pull/1447/files#diff-35c9049ac0d019beca1d12aeed7ff3d77812d685e1497daf41c19c6a02b5e619R394) added.